README.md

```diff
@@ -90,7 +90,7 @@ Associated View `HomeView.java`:
 public class HomeView extends BaseScreenView<HomeScreen> {
   public HomeView(Context context) {
     super(context);
-    inflate(R.layout.home);
+    inflate(context, R.layout.home, this);
```

Comment:
the version without context will work as soon as we release the next version, but I agree it's confusing right now

Reply:
Ok, that's the only way I could get it to compile.